docs/how-to-deploy-scalar-manager.md

```diff
@@ -24,23 +24,23 @@ grafana:
 If you already have a deployment of kube-prometheus-stack, please upgrade the configuration using the following command.
 
 ```console
-helm upgrade <release name> prometheus-community/kube-prometheus-stack -n <namespace> -f /path/to/<your custom values file for kube-prometheus-stack>
+helm upgrade <RELEASE_NAME> prometheus-community/kube-prometheus-stack -n <NAMESPACE> -f /path/to/<YOUR_CUSTOM_VALUES_FILE_FOR_KUBE_PROMETHEUS_STACK> --version <CHART_VERSION>
```

Comment:
Since the user should also change `path/to`, how about we change this to `/<PATH_TO_YOUR_CUSTOM_VALUES_FILE_FOR_KUBE_PROMETHEUS_STACK>`?

```suggestion
helm upgrade <RELEASE_NAME> prometheus-community/kube-prometheus-stack -n <NAMESPACE> -f /<PATH_TO_YOUR_CUSTOM_VALUES_FILE_FOR_KUBE_PROMETHEUS_STACK> --version <CHART_VERSION>
```

Reply:
It looks good to me!
I will update all documents. Thank you!